pynumaflow/sourcetransformer/_dtypes.py

```diff
@@ -12,6 +12,15 @@
 
 @dataclass(init=False)
 class Message:
+    """
+    event_time_to_drop 1969-12-31 00:00:00 +0000 UTC is set to be slightly
+    earlier than Unix epoch -1 (1969-12-31 23:59:59.999 +0000 UTC)
+    As -1 is used on Numaflow to indicate watermark is not available, event_time_to_drop is
+    used to indicate that the message is dropped hence, excluded from watermark calculation
+    """
+
+    event_time_to_drop = datetime(1969, 12, 31, 0, 0, 0, tzinfo=timezone.utc)
```

Comment:
Do we really need this as a new field? Can't we just store this as a constant, and assign?